lib/yggdrasil/GuardianErrorHandler.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.GuardianErrorHandler do
+  @behaviour Guardian.Plug.ErrorHandler
+
+  import Plug.Conn
+
+  def unauthenticated(conn, _params) do
+    conn
+      |> send_resp(:unauthorized, "")
```

Comment:
It does take an integer so we could replace with 401 instead if you think that would be clearer. 

Sent from my iPhone

> On Jan 9, 2016, at 9:58 AM, Sean Copenhaver <notifications@github.com> wrote:
> 
> In lib/yggdrasil/GuardianErrorHandler.ex:
> 
> > @@ -0,0 +1,15 @@
> > +defmodule Yggdrasil.GuardianErrorHandler do
> > +  @behaviour Guardian.Plug.ErrorHandler
> > +
> > +  import Plug.Conn
> > +
> > +  def unauthenticated(conn, _params) do
> > +    conn
> > +      |> send_resp(:unauthorized, "")
> Ah that makes sense. Might want to leave a comment saying that you gotta use :unauthorized so the user will get a 401.
> 
> —
> Reply to this email directly or view it on GitHub.
>